app/controllers/events_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: false
+
+# class Events
+class EventsController < ApplicationController
+  def new
+    @event = Event.new
+  end
+
+  def create
+    @event = Event.new(events_paramas)
+
+    if @event.save
+      redirect_to '/views/events/index'
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def index
+    @event = Event.all
+  end
+
+  private
+
+  def events_paramas
+    params.require(:event).permit(:Titulo, :Descripcion, :Fecha, :Ubicacion, :Costo)
```

Comment:
The fields in a table must be lowercase.
You can run a migration to rename this fields.

See the documentation at:
https://edgeguides.rubyonrails.org/active_record_migrations.html
https://edgeapi.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-rename_column